modules/java/test/android_test/settings.gradle

```diff
@@ -0,0 +1,6 @@
+rootProject.name = 'android_test'
+
+include ':opencv'
+project(':opencv').projectDir = new File('D:\\Android\\opencv-4.9.0-android-sdk_orig\\OpenCV-android-sdk\\sdk')
```

Comment:
I believe it should be cmake template. Reference to build/opencv-abdroid/sdk should be placed here.